apps/gro/fields/address.js

```diff
@@ -2,7 +2,8 @@
 
 module.exports = {
   'address-textarea': {
-    validate: ['required'],
+    validate: ['required',
+      {'type': 'regex', 'arguments': [/^[a-zA-Z0-9\s,-]*$/]}],
```

Comment:
couldn't we use `\w` instead of `a-zA-Z0-9`? `\w` includes `_` though so not sure if we want to allow this, ignore me if we want to explicitly omit